apps/src/utils.js

```diff
@@ -289,6 +289,7 @@ export function getPegasusHost() {
       var name = window.location.hostname.split('.')[0];
       switch (name) {
         case 'localhost':
+        case 'localhost-studio':
```

Comment:
Thanks for calling this out Brent, droplet block documentation is fixed now too for localhost-studio.code.org 🎉